ibm/service/power/resource_ibm_pi_volume_group.go

```diff
@@ -131,20 +158,20 @@ func resourceIBMPIVolumeGroupRead(ctx context.Context, d *schema.ResourceData, m
 		return diag.FromErr(err)
 	}
 
-	client := st.NewIBMPIVolumeGroupClient(ctx, sess, cloudInstanceID)
+	client := instance.NewIBMPIVolumeGroupClient(ctx, sess, cloudInstanceID)
 
 	vg, err := client.GetDetails(vgID)
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
-	d.Set("volume_group_id", vg.ID)
-	d.Set("volume_group_status", vg.Status)
-	d.Set("consistency_group_name", vg.ConsistencyGroupName)
-	d.Set("replication_status", vg.ReplicationStatus)
-	d.Set(PIVolumeGroupName, vg.Name)
+	d.Set(Arg_VolumeGroupName, vg.Name)
 	d.Set(Arg_VolumeIDs, vg.VolumeIDs)
-	d.Set("status_description_errors", flattenVolumeGroupStatusDescription(vg.StatusDescription.Errors))
+	d.Set(Attr_ConsistencyGroupName, vg.ConsistencyGroupName)
+	d.Set(Attr_ReplicationStatus, vg.ReplicationStatus)
+	d.Set(Attr_StatusDescriptionErrors, flattenVolumeGroupStatusDescription(vg.StatusDescription.Errors))
```

Comment:
Can you please add a null check for `vg.StatusDescription`?

Reply:
made changes